src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts

```diff
@@ -147,6 +151,12 @@ export class RgwMultisiteDetailsComponent implements OnDestroy, OnInit {
     }
   }
 
+  openMultisiteSetupWizard() {
+    this.bsModalRef = this.modalService.show(RgwMultisiteWizardComponent, {
+      size: 'lg'
+    });
+  }
```

Comment:
i don't prefer opening a basic modal for doing a complex multi-step wizard. especially when its not routed which means any click outside the modal can simply close down the modal. Can you make this modal routed? its a simple process, I think @afreen23 is using that on the [nvmeof PRs](https://github.com/ceph/ceph/pull/58241/files#diff-f1b07e24a23214549e4689888989cab3637a856cb7f75cb771364af0cf2545bbR65)

Reply:
Done. Thanks @nizamial09